components/RecipeCard/RecipeCard.tsx

```diff
@@ -0,0 +1,26 @@
+import Image from 'next/image';
+import { RecipeMeta } from '../../lib/types';
+import { Card, Category, Clock, Content, Time, Title } from './RecipeCard.styles';
+
+export type RecipeCardProps = {
+  meta: RecipeMeta;
+};
+
+export const RecipeCard = ({ meta }: RecipeCardProps) => {
+  const { category, title, time, image } = meta;
+  const src = require(`../../_recipes/images/${image}`);
```

Comment:
Nit: This way `webpack` will bundle all image paths to the JavScript bundle. Its ok then we have 4 images, but if we have 1000 images it will be quite a large chunk of JavaScript.

Reply:
Yeah, there are probably a few different ways to approach the images in a submodule. Next is opinionated about where images can live, and this was the simplest solution I found in the short term.